### PR TITLE
update material2 branch

### PR DESCRIPTION
**Pull request by dweitz43**
* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
branch update

* **What is the current behavior?** (You can also link to an open issue here)
material2 branch had fallen way behind master branch.

* **What is the new behavior (if this is a feature change)?**
material2 branch updated to master branch and rc.5, MdModule created for material2 components.

* **Other information**:
thanks to @spock123 for the help in successfully integrating material2!

### CODE REVIEW

File: src/vendor.browser.ts [hunk 1]
```diff
@@ -12,9 +14,36 @@ import '@angular/forms';
 import '@angular/http';
 import '@angular/router';
 
+// Angular2 Material
+import '@angular2-material/core';
```
Comment (binarious) [inline src/vendor.browser.ts]: Why this? I did not include this in my update and it works: https://github.com/Sirimangalo/meditation-plus-angular/commit/824c60bd2dbb4063e869920f13e93539af3d453d

Comment (dweitz43) [inline src/vendor.browser.ts]: @binarious i realize now that the material components are also being included in the main bundle of the production build as well, don't know how i missed in the first place...thanks for the tip, i am updating the pr now

**Comment by dweitz43** [comment]
@gdi2290 is there a specific reason why this cannot be merged? Am I missing something in the material integration?

**Comment by gdi2290** [comment]
LGTM

**Comment by gdi2290** [comment]
thanks!

**Comment by dweitz43** [comment]
@gdi2290 thanks a lot!

**Comment by kodeine** [comment]
update material branch to support angular rc6?

**Pull request merged**